test_requirements.in

```diff
@@ -0,0 +1,5 @@
+coverage==5.0.3
+pytest==5.3.5
+pytest-black==0.3.8
+pytest-flake8==1.0.4
+pytest-cov==2.8.1
```

Comment:
We don't actually need those requirements when running the container "in production".

Reply:
Ah wait, we need them to run the CI tests :/